content/concepts/introduction/overview.md

```diff
@@ -1,160 +1,143 @@
+"[peers](../fundamentals/peers.md)."
+
+### Peer-to-peer basics
+
+P2P networks are decentralized, meaning participants communicate directly with
+one another on a relative "equal footing." No central server or
+authority controls the network.
+P2P networks do not require a privileged set of "servers" that behave differently
+from their "clients," as in the predominant
+[client-server model](https://en.wikipedia.org/wiki/Client%E2%80%93server_model).
+
+P2P networks can take many forms, including file-sharing systems like
+[BitTorrent](https://www.bittorrent.com/), blockchain networks like [Bitcoin](https://bitcoin.org/en/)
+and [Ethereum](https://ethereum.org/en/), and decentralized communication standards like
+[Matrix](https://matrix.org/). These systems all have different challenges and tradeoffs,
+but they share the goal of improving upon the traditional client-server networking model.
+
+### Background of libp2p
+
+libp2p was initially developed as part of the [InterPlanetary File System (IPFS)](https://ipfs.tech/)
+project as its wire protocol but has since phased out into a networking stack that has been adopted
+by a wide range of other projects as a networking layer. It provides a set of specifications that
+can be adapted to support various protocols, allowing libp2p applications to operate in diverse
+runtimes and networking environments.
+
+> Discovering and connecting with other peers is a key challenge in P2P networking. In the past,
+> each P2P application had to develop its own solution for this problem, leading to a lack of
+> reusable, well-documented P2P protocols. IPFS looked to existing research and networking
+> applications for inspiration, but found few code implementations that were usable and adaptable.
+> Many of the existing implementations had poor documentation, restrictive licensing, outdated code,
+> no point of contact, were closed source, deprecated, lacked specifications, had unfriendly APIs,
+> or were tightly coupled with specific use cases and not upgradeable. As a result, developers often
+> had to reinvent the wheel each time they needed P2P protocols, rather than being able to reuse
+> existing solutions.
+
+{{< alert icon="" context="info">}}
+libp2p was designed to address the limitations of traditional P2P networking approaches and these
+existing network models, with the goal of enabling the distributed web.
+{{< /alert >}}
+
+## Why libp2p?
+
+There are several reasons to consider using libp2p as a networking layer to create a robust P2P application:
+
+- **Modularlity**: libp2p is designed to be modular, allowing developers to mix and match different components
+  to meet the needs of their particular application. This makes it easy to customize the networking stack
+  to fit the specific requirements of any P2P application.
+
+  - **Transport agnosticism**: libp2p provides a set of specifications that can be adapted to support various
```

Comment:
```suggestion
- **Extensive transport configurability**: libp2p provides a set of specifications that can be adapted to support various
```
I think this should not be categorized as transport agnostic, because that seems like the transport utilized is completely opaque to libp2p and the user. When in fact, configurability of different transports is the selling point.

Reply:
that's true!